apps/full-site-editing/full-site-editing-plugin/dotcom-fse/plugins/close-button-override/style.scss

```diff
@@ -11,7 +11,8 @@
 	display: flex;
 	align-items: center;
 	padding: 9px 10px;
-	margin-left: -8px;
+	margin-left: -24px;
```

Comment:
I think it's fine to optimize for the new version and have the old version be aligned a bit off, especially since there might not be a great way to retroactively apply these styles only if using the old version.

Reply:
Added backward support in b88c7e9900fc9e8dc5076ddb65b052d71d28a2cb